src/app/ecosystem-explorer/project-form/page.tsx

```diff
@@ -0,0 +1,78 @@
+'use client'
+
+import { useState } from 'react'
+
+import { Button } from '@/components/Button'
+
+import { submitProjectToGithub } from './services/github'
+import {
+  ALLOWED_IMAGE_FORMATS,
+  getFileFormat,
+  convertToBase64,
+} from './services/github/utils'
+
+export default function ProjectForm() {
+  const [files, setFiles] = useState<FileList | null>()
+  const [projectName, setProjectName] = useState<string | undefined>('')
+
+  const [link, setLink] = useState<string | undefined>('')
```

Comment:
Are these the right types for these?

Reply:
This was just a dummy page to test the integration. I will remove it when the PR is ready to merge.